draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -152,6 +153,7 @@ update is made to the catalog zone's contents as per serial number arithmetic
 defined in [@!RFC1982].  Otherwise, secondary nameservers might not notice
 updates to the catalog zone's contents.
 
+[FIXME: this ignores that the negative TTL is min(SOA TTL, SOA MINIMUM). I also don't think this argument for zero is very strong.]
```

Comment:
Hehe! Indeed let's be compatible with that other draft at least ;).
I propose to remove the entire paragraph below.

Reply:
:+1: